remote-access/samba.md

```diff
@@ -105,3 +105,25 @@ workgroup = <your workgroup name here>
 ```
 
 That should be enough to share the folder. On your Windows device, when you browse the network, the folder should appear and you should be able to connect to it.
+
+### "Host is down" error
```

Comment:
It probably makes sense to move this section to below the "Mount the folder on the Raspberry Pi" section rather than below the "Sharing a folder for use by Windows" section?